### PR TITLE
Bug fix/1879

### PR DESCRIPTION
**Pull request by skrebetsnataliia**
https://github.com/ita-social-projects/GreenCity/issues/1879
https://github.com/ita-social-projects/GreenCity/issues/1883
https://github.com/ita-social-projects/GreenCity/issues/1890
https://github.com/ita-social-projects/GreenCity/issues/1904

prveous:
image
image
image
1024 px
image
image
768px
image
image
1200px
image
575px
image

results:
image
image
image

1024px
image
image
768px
image
image
1200px
image
575px
image

### CODE REVIEW

File: src/app/component/eco-news/components/news-list/news-list.component.scss [hunk 1]
```diff
@@ -78,7 +84,8 @@ a {
 
 @media screen and (max-width: 1199px) {
   .wrapper {
-    max-width: 960px;
+    //max-width: 960px;
```
Comment (luchward) [inline src/app/component/eco-news/components/news-list/news-list.component.scss:87]: You forgot to remove

File: src/app/component/eco-news/components/news-list/news-list.component.scss [hunk 2]
```diff
@@ -88,7 +95,8 @@ a {
 
 @media screen and (max-width: 991px) {
   .wrapper {
-    max-width: 740px;
+    //max-width: 740px;
```
Comment (luchward) [inline src/app/component/eco-news/components/news-list/news-list.component.scss:98]: The same thing

**Review by luchward — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

File: src/app/component/eco-news/components/news-list/news-list.component.scss (hunk 1 above)
Comment (skrebetsnataliia) [reply src/app/component/eco-news/components/news-list/news-list.component.scss]: I've deleted

File: src/app/component/eco-news/components/news-list/news-list.component.scss (hunk 2 above)
Comment (skrebetsnataliia) [reply src/app/component/eco-news/components/news-list/news-list.component.scss]: I've deleted

**Pull request merged**